package.json

```diff
@@ -35,22 +52,27 @@
     "@babel/runtime": "^7.17.9",
     "babel-eslint": "^10.1.0",
     "babel-node": "^0.0.1-security",
+    "babel-register": "^6.26.0",
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.20.0",
+    "chai": "^4.3.6",
+    "chai-http": "^4.3.0",
     "cors": "^2.8.5",
     "cross-env": "^7.0.3",
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
+    "joi": "^17.6.0",
+    "jsonwebtoken": "^8.5.1",
     "pg": "^8.7.3",
     "pg-hstore": "^2.3.4",
-    "sequelize": "^5.21.6",
-    "swagger-ui-express": "^4.4.0"
```

Comment:
why did you remove this dependency

Reply:
and also docker app is not working as expected

![Screenshot from 2022-05-29 20-33-48](https://user-images.githubusercontent.com/59834399/170886239-9ff981a4-4935-4aea-af1c-93c7b9991be9.png)
d

Reply:
same issue applies to this [PR](https://github.com/atlp-rwanda/bravo-bn-be/pull/11)